src/show/components/Region/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const regions = [
+  'Choose',
+  'United Kingdom',
+  'United States',
+];
+
+const Region = ({ inputs, handleInputChange, setInputs }) => {
```

Comment:
how this component is global , if i need to use this dropdown somewhere else with different array , then how will i pass the values here . create an global component for dropdown and name it dropdown

Reply:
done